packages/core/src/external/aws/document-signing/document-bulk-signer-lambda.ts

```diff
@@ -0,0 +1,31 @@
+import * as AWS from "aws-sdk";
+import { makeLambdaClient } from "../lambda";
+import { DocumentBulkSigner, DocumentBulkSignerRequest } from "./document-bulk-signer";
+
+export class DocumentBulkSignerLambda extends DocumentBulkSigner {
+  private lambdaClient: AWS.Lambda;
+
+  constructor(region: string, readonly lambdaName: string) {
+    super(region);
+    this.lambdaClient = makeLambdaClient(this.region);
+  }
+
+  async sign({ patientId, cxId, requestId }: DocumentBulkSignerRequest) {
+    const payload: DocumentBulkSignerRequest = {
+      patientId: patientId,
+      cxId: cxId,
+      requestId: requestId,
+    };
+
+    this.lambdaClient
+      .invoke({
+        FunctionName: this.lambdaName,
+        InvocationType: "RequestResponse",
+        Payload: JSON.stringify(payload),
+      })
+      .promise()
+      .catch(error => {
```

Comment:
.catch on the promise

Reply:
The lambda itself handles errors and sends a failed documentSigning back to the internal endpoint, so all this is really gonna catch is if the lambda name is wrong. I am just going to have a console.log here in the catch and not rethrow any error